integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -156,8 +156,9 @@ <h3 class="heading font-bold text-black">
                         </div>
                         <div>
                             <label>{% trans 'Position' %}</label>
-                            <div class="help-text">
-                                {% trans 'If you leave these fields blank, they are automatically derived from the address.' %}
+                            <div>
+                                <input type="checkbox" id="auto-fill-coordinates">
+                                <label for="auto-fill-coordinates">{% trans 'Calculate coordinates automatically from address' %}</label>
                             </div>
```

Comment:
```suggestion
                            </div>
                            <div id="nominatim-error" class="bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 mb-5 hidden" role="alert"></div>

```
I'd also show an error if anything went wrong with fetching.